.github/workflows/ci.yml

```
@@ -0,0 +1,48 @@
+      - name: Flake checks (no system builds)
+        run: nix flake check --keep-going --print-build-logs
+
+  eval-hosts:
+    runs-on: macos-15
+    needs: flake-check
+    strategy:
+      matrix:
+        host: [higgins, benson, spaceblack, dunston]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: DeterminateSystems/nix-installer-action@v19  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)